renpy/common/00defaults.rpy

```diff
@@ -58,6 +58,9 @@ init -1500 python:
 
     # If not None, the default value of set_volume (sfx)
     config.default_sfx_volume = 1.0
+    
+    # The default values of set_volume for custom mixers
+    config.default_mixer_volume = {}
```

Comment:
Might make sense to deprecate old mixer configs (default them to `None`). If they are set (non-`None`), add their values to the new `default_mixer_volume`, otherwise just ignore.

Reply:
I think you forgot to add the default values to `default_mixer_volume` for those 3 builtin mixers.

Reply:
I assume they are not needed because the mixers are set to 1.0 by default:
https://github.com/renpy/renpy/blob/master/renpy/preferences.py#L323